tests/cli/test_compose_alibaba.sh

```diff
@@ -0,0 +1,197 @@
+#!/bin/bash
+# Note: execute this file from the project root directory
+
+#####
+#
+# Make sure we can build an image and deploy it inside Alibaba cloud!
+#
+#####
+
+set -e
+
+. /usr/share/beakerlib/beakerlib.sh
+. $(dirname $0)/lib/lib.sh
+
+CLI="${CLI:-./src/bin/composer-cli}"
+
+
+rlJournalStart
+    rlPhaseStartSetup
+        if [ -z "$ALICLOUD_ACCESS_KEY" ]; then
+            rlFail "ALICLOUD_ACCESS_KEY is empty!"
+        else
+            rlLogInfo "ALICLOUD_ACCESS_KEY is configured"
+        fi
+
+        if [ -z "$ALICLOUD_SECRET_KEY" ]; then
+            rlFail "ALICLOUD_SECRET_KEY is empty!"
+        else
+            rlLogInfo "ALICLOUD_SECRET_KEY is configured"
+        fi
+
+        ALICLOUD_BUCKET="${ALICLOUD_BUCKET:-composer-test}"
+        ALICLOUD_REGION="${ALICLOUD_REGION:-us-east-1}"
+
+        rlLogInfo "ALICLOUD_BUCKET=$ALICLOUD_BUCKET"
+        rlLogInfo "ALICLOUD_REGION=$ALICLOUD_REGION"
+
+        for package in jq; do
+            if ! rlCheckRpm "$package"; then
+                rlRun -t -c "dnf -y install $package"
+                rlAssertRpm "$package"
+            fi
+        done
+
+        ALI_DIR=`mktemp -d /tmp/alicloud.XXXXX`
+        # use the CLI b/c Ansible modules are not yet upstream and are unreliable
+        TAR_FILE="aliyun-cli-linux-3.0.32-amd64.tgz"
+        curl -L https://github.com/aliyun/aliyun-cli/releases/download/v3.0.32/$TAR_FILE > $ALI_DIR/$TAR_FILE
+        tar -C $ALI_DIR/ -xzvf $ALI_DIR/$TAR_FILE
```

Comment:
I'd suggest to download and use the latest version here as well (as in `remove_old_objects_alibaba.sh`).